### PR TITLE
creating search bar

### PR DESCRIPTION
**Pull request by mykola-chernysh**
[DEMO LINK](https://mykola-chernysh.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://mykola-chernysh.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+.search-line {
+  width: 100%;
+  height: 70px;
+  padding: 0 0 0 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-family: Avenir, Helvetica, sans-serif;
```
Comment (valerii-derkach) [inline src/style.css:51]: and here

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 137.5%;
+  color: #3d4e61;
+}
+
+.search-line::placeholder {
+  font-family: Avenir, Helvetica, sans-serif;
```
Comment (valerii-derkach) [inline src/style.css:59]: same here

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  padding: 0 0 0 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 137.5%;
```
Comment (valerii-derkach) [inline src/style.css:54]: use integers

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  color: #3d4e61;
+}
+
+.search-line::placeholder {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 137.5%;
```
Comment (valerii-derkach) [inline src/style.css:62]: use integers

**Review by valerii-derkach — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -11,6 +11,42 @@
...
+      data-qa="big"
+      class="search-form"
+      action="/"
+      method="post"
+    >
+      <img
+        src="images/Search.svg"
+        alt="icon-search"
+        class="search-image"
+        >
```
Comment (marina-tilniak) [inline src/index.html:24]:
```suggestion
        class="search-image"
      >
```

**Review by marina-tilniak — APPROVED** [review]

**Pull request closed without merging**